.env.example

```
@@ -0,0 +1 @@
+DATABASE_URL="postgresql://USERNAME:PASSWORD@localhost:6500/DATABASENAME?schema=public"
```

Better to just remove the value entirely

```suggestion
DATABASE_URL=
```